src/components/stepforms/schemas/replaceText.ts

```diff
@@ -0,0 +1,31 @@
+export default {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Replace text step',
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      enum: ['replacetext'],
+    },
+    searchColumn: {
+      type: 'string',
+      minLength: 1,
+      title: 'Search',
+      description: 'Column in which to replace text',
+      attrs: {
+        placeholder: 'Enter a column',
+      },
+    },
+    oldStr: {
+      type: 'string',
+      description: 'String to replace',
+      minLength: 1,
```

Comment:
Why min length? If I want to replace empty strings with something else

Reply:
Then you'd use the normal `replace` step, which replaces values

Reply:
I don't know the codebase as well as you but from a user perspective it makes no sense to have two different steps if I want to replace '' or 'a'

Reply:
At first, we were wondering if replacing text and values should both be done in the same step or not. After discussion with product, we've decided to to it in two separate steps.

The difference between the two steps is:

* I want to replace all values in column `col` containing the literal value `"a"` with `42` -> that's a `replace`, the type can change. Only cells with `"a"` in them will be modified -> Cells with "baba" are left untouched.
* In my string columns, I want to replace the string "guys" with "people" -> That's a `replacetext`. All cells containing "guys" will be modified: `"guys"` will become `"people"`, `"hello guys"` will become `"hello people"`. 

In your example the expeected behaviour of a `replacetext("", "a")` would be `"foo"` -> `"afaoaoa"`, which is probably not what the end user wants

Reply:
Ok got it thanks a lot

Reply:
Too bad it wasn't called "replace substring"
I didn't realize what this would do until I read your last example 😅

Reply:
Yes, naming is a bit frustrating for devs, but with product we thought that it was better to have "text" everywhere rather than let end users figure out by themselves wtf a "substring" is supposed to be :smile: